e2e_tests/test_shared_services.py

```diff
@@ -91,6 +92,7 @@ async def test_patch_firewall(verify):
 ]
 
 
+@pytest.mark.extended
```

Comment:
this currently fails 100% of the time on the main branch and is very likely to fail on repeat runs at some point in PR environments.

Reply:
Yeah it still seems to be failing with the same problem "secret being recovered and can't be deleted". I will mark this test as skipped for now.